actiontext/test/generators/action_text_generator_test.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "minitest/mock"
+require "generators/action_text/install/install_generator"
+
+class ActionTextGeneratorTest < Rails::Generators::TestCase
+  destination File.expand_path("../../tmp", __dir__)
+  setup :prepare_destination
+  tests ActionText::Generators::InstallGenerator
+
+  def destination_root
+    tmp_path "foo_bar"
+  end
+
+  def tmp_path(*args)
+    @tmp_path ||= File.expand_path("../../tmp", __dir__)
+    File.join(@tmp_path, *args)
+  end
+
+  def sample_app_path
+    tmp_path("foo_bar")
+  end
+
+  def rails(cmd)
+    `#{Gem.ruby} #{RAILS_FRAMEWORK_ROOT}/railties/exe/rails #{cmd}`
+  end
+
+  def bundled_rails(cmd)
+    `bundle exec rails #{cmd}`
+  end
+
+  def generate_sample_app
+    FileUtils.rm_rf(sample_app_path)
+    FileUtils.mkdir_p(sample_app_path)
+
+    quietly {
+      rails("new #{sample_app_path} --skip-action-mailer --skip-action-cable --skip-sprockets --skip_bootsnap --skip_webpack_install --quiet")
+    }
+  end
```

Comment:
How much of this is copied from other generator tests? Seems like there's quite a bit of setup here.

Reply:
Almost everything in the setup from this generator: https://github.com/rails/rails/blob/master/railties/test/railties/generators_test.rb

Tried to require the `isolations/abstract_unit` and use `generators_test_helper` but faced a lot of issues. That was 4 days ago so can't recall exactly what the errors was but I can try it again and send it here.

Reply:
Yeah, we can’t duplicate the entire setup 👍

Reply:
@kaspth Removed the duplication and moved the test cases to railties. Could you please review the change now?